src/components/form/Input.js

```diff
@@ -1,31 +1,32 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import { Field } from 'react-final-form'
-import FormControl from '@material-ui/core/FormControl'
 import { default as MUIInput } from '@material-ui/core/Input'
```

Comment:
If `Input` is the default export from `@material-ui/core/Input`, then I think you can already name it whatever you'd like, i.e. `import MUIInput from '@material-ui/core/Input'`